apps/explorer/src/pages/checkpoints/CheckpointDetail.tsx

```diff
@@ -17,17 +17,19 @@ import { Text } from '~/ui/Text';
 import { GROWTHBOOK_FEATURES } from '~/utils/growthbook';
 
 function CheckpointDetail() {
-    const { digest } = useParams<{ digest: string }>();
-    const rpc = useRpcClient();
+    const { id } = useParams<{ id: string }>();
+    const parsed = parseInt(id!);
```

Comment:
is there a scenario where we successfully parse a digest as a number instead of `NaN`?

e..g `parseInt("0xdfddhu42323") => 59309`

Reply:
Pass radix of 10.